public/css/lanyon.css

```diff
@@ -156,7 +156,7 @@ h1, h2, h3, h4, h5, h6 {
   width: 14rem;
   visibility: hidden;
   overflow-y: auto;
-  font-family: "PT Sans", Helvetica, Arial, sans-serif;
+  font-family: 'Noto Serif', serif;
```

Comment:
If you added the @import from https://fonts.google.com/earlyaccess it should work. I can test your PR tonight.